setup.py

```diff
@@ -40,6 +40,7 @@ def find_version(*file_paths):
         "msal",
         "requests",
         "numpy",
+        "anthropic"
```

Comment:
you are missing a comma at the end of this line

Reply:
Fixed 😄